manifests/h/HiroSystems/Clarinet/1.4.2/HiroSystems.Clarinet.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: HiroSystems.Clarinet
+PackageVersion: 1.4.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
The package installs normally, but needs this dependency to run:
![image](https://user-images.githubusercontent.com/114952565/217415677-330e8bbe-1428-4458-b612-1a2ab3c6fded.png)